Source/Extensions/Blazorise.SignaturePad/SignaturePad.razor.cs

```diff
@@ -196,7 +196,7 @@ public async Task NotifyEndStroke( string dataUrl, double offsetX, double offset
     /// <param name="data">Byte array that holds the image data.</param>
     /// <param name="imageType">Image type.</param>
     /// <returns>Data url.</returns>
-    private static string GetDataUrl( byte[] data, SignaturePadImageType imageType )
+    public static string GetDataUrl( byte[] data, SignaturePadImageType imageType )
```

Comment:
Can we know reuse ourselves in the examples / demos instead of manually building the strings?